addons/api/addon/generated/models/auth-method.js

```diff
@@ -101,9 +101,79 @@ export default class GeneratedAuthMethodModel extends BaseModel {
   })
   idp_ca_certs;
 
-  @attr('account-claim-map-array', {
+  @attr('account-value-map-array', {
     emptyArrayIfMissing: true,
     isNestedAttribute: true,
   })
   account_claim_maps;
+
```

Comment:
Can we add a comment here saying these attributes are for LDAP?

Reply:
@DhariniJeeva yes good call